### ISSUE TITLE
readme file 

### ISSUE THREAD

**Issue by rahul-0000-code** [naushaddhun/Indoor-Navigation#2]
Can you add a seperate readme file in which whole process for cloning and process can  be read

**Comment by naushaddhun** [comment]
Hey Rahul,

This repository is targeted towards a very specific use-case that we had for our pre-final year project. This repository has not been updated since 5 years and I am not sure which Unity version has been used, maybe I can check and let you know if this is what you need. Else if I can be of any other help regarding the same topic, I am happy to help you out, you can reach out to me directly.

**Comment by rahul-0000-code** [comment]
Yes bro or else you can pin me a tutorial or a guided path that could be much better
